### PR TITLE
Scott renz

### PR DESCRIPTION
**Pull request by scottrenz**

### CODE REVIEW

File: src/03_modules.py [hunk 1]
```diff
@@ -10,22 +10,24 @@
 
 # Print out the command line arguments in sys.argv, one per line:
 # YOUR CODE HERE
-
+[print(arg) for arg in sys.argv]
```
Comment (brianilles) [inline src/03_modules.py:13]: Great work here Scott. Using an LC here indicates your high level

File: src/04_printing.py [hunk 2]
```diff
@@ -11,7 +11,16 @@
...
+print('{}'.format(x) )
+print('{0:.2f}'.format(y))
+print('{}'.format(z))
 
-# Finally, print the same thing using an f-string
\ No newline at end of file
+# Finally, print the same thing using an f-string
+print(f"{x}")
+print(f"{y:.2f}" )
+print(f"{z}")
```
Comment (brianilles) [inline src/04_printing.py]: Good job. We can avoid using the print statement so many times by using commas between each expression. For instance, with the f-string, like so `print(f'x is {x}, y is {y}, z is {z}')`

File: src/07_slices.py [hunk 3]
```diff
@@ -12,26 +12,26 @@
...
-print()
+print(a[2:4])
 
 # Output every element except the first one: [4, 1, 7, 9, 6]
-print()
+print(a[1:])
 
 # Output every element except the last one: [2, 4, 1, 7, 9]
-print()
+print(a[:-1])
```
Comment (brianilles) [inline src/07_slices.py:31]: Great job with the negative indexes here

File: src/10_functions.py [hunk 4]
```diff
@@ -1,12 +1,18 @@
 # Write a function is_even that will return true if the passed-in number is even.
 
 # YOUR CODE HERE
-
+def is_even(n):
+    if int(n)/2 == int(int(n)/2):
+        return True
```
Comment (brianilles) [inline src/10_functions.py]: Good job. We can clean this up by simply having `is_even` return `n % 2 == 0` which makes the return value a bool

File: src/14_cal.py [hunk 5]
```diff
@@ -21,4 +21,20 @@
...
+if l == 3:
+  y = s[2]
+if l > 1:
+  m = s[1]
+try:
+    str = c.formatmonth(int(y), int(m))
+except:
+  f = open('instruct.txt','r')
+  str = f.read()
+print(str)
```
Comment (brianilles) [inline src/14_cal.py:40]: Great job! Have you used calendar and date time before?

**Review by brianilles — COMMENTED** [review]
Incredible start to the project!

**Comment by scottrenz** [comment]
No, I never used calendar and date time before.
I did a lot of googling to get it working.

File: src/10_functions.py [hunk 6]
```diff
@@ -2,8 +2,7 @@
 
 # YOUR CODE HERE
 def is_even(n):
-    if int(n)/2 == int(int(n)/2):
-        return True
+    return int(n)/2 == int(int(n)/2)
```
Comment (brianilles) [inline src/10_functions.py:5]: Great work. We can also simply return ` return n % 2 == 0` here. Both work however!

File: src/15_classes.py [hunk 7]
```diff
@@ -2,36 +2,41 @@
...
 # `size`, `lat`, and `lon` to the constructor. What should it inherit from?
 
 # YOUR CODE HERE
+class Geocache(Waypoint):
+    def __init__(self, name, difficulty, size, lat, lon):
+        super().__init__(name, lat, lon)
+        self.difficulty = difficulty
+        self.size = size
+    def __str__(self):
+        return f'\n\tGeocache\nName:\t\t{self.name}\nDifficulty:\t{self.difficulty}\nSize:\t\t{self.size}\nLatitude:\t{self.lat}\nLongitude:\t{self.lon}\n'
```
Comment (brianilles) [inline src/15_classes.py:35]: Love the way you format this.

File: src/prime.py [hunk 8]
```diff
@@ -0,0 +1,15 @@
...
+    r = num % v
+    if r == 0:
+        break
+print(num,r,v)
+
+if v == num -1 or (r == 0 and v == 0):
+    print(f'{num} is prime')
+else:
+    print(f'{num} is not prime')
+
```
Comment (brianilles) [inline src/prime.py:15]: Great work with the stretch. Code is readable!

**Review by brianilles — APPROVED** [review]
Excellent work on today's project Scott! Its evident you have a built a strong base in Python in a short period of time. Approving and merging this PR!

**Pull request merged**